src/main/java/com/linkedin/replica/serachEngine/databaseHandlers/impl/ArangoHandler.java

```diff
@@ -0,0 +1,151 @@
+package com.linkedin.replica.serachEngine.databaseHandlers.impl;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import com.arangodb.ArangoCursor;
+import com.arangodb.ArangoDB;
+import com.arangodb.util.MapBuilder;
+import com.linkedin.replica.serachEngine.config.Configuration;
+import com.linkedin.replica.serachEngine.config.DatabaseConnection;
+import com.linkedin.replica.serachEngine.databaseHandlers.DatabaseHandler;
+import com.linkedin.replica.serachEngine.models.Company;
+import com.linkedin.replica.serachEngine.models.Job;
+import com.linkedin.replica.serachEngine.models.User;
+import com.linkedin.replica.serachEngine.models.Post;
+
+/**
+ * Implementation of ArangoHandler which is responsible for serving specific request from ArangoDB
+ */
+public class ArangoHandler implements DatabaseHandler{
+	private Properties properties;
+	private ArangoDB arangoDB;
+	
+	public ArangoHandler() throws FileNotFoundException, ClassNotFoundException, IOException, SQLException{
+		arangoDB = DatabaseConnection.getInstance().getArangodb();
+		properties = new Properties();
+		properties.load(new FileInputStream(Configuration.getInstance().getArangoNamesConfigPath()));
+	}
+	
+	/**
+	 * Search for users 
+	 * @return
+	 * 	return list of users
+	 */
+	public List<User> searchUsers(String name) {
+		String dbName = properties.getProperty("db.name");
+		String collectionName = properties.getProperty("collection.user.name");		
+		
+		// lowerCase name to avoid case sensitive search
+		name = name.toLowerCase();
+		
+		// query for getting user if there is a match of his/her firstName or lastName with input
+		String query = "FOR t IN "+ collectionName +" FILTER "
+				+ "LOWER(t.firstName) LIKE @name || "
+				+ "LOWER(t.lastName) LIKE  @name "
+				+ "RETURN t";
+		
+		// bind variables in query
+		Map<String, Object> bindVars = new MapBuilder().put("name", "%"+name+"%").get();
```

Comment:
Can you please adjust the indentation in this file? Make sure that the whitespaces consist of only tabs not spaces